source/adapter.js

```diff
@@ -254,13 +254,71 @@ AdapterJS.maybeFixConfiguration = function (pcConfig) {
 AdapterJS.addEvent = function(elem, evnt, func) {
   if (elem.addEventListener) { // W3C DOM
     elem.addEventListener(evnt, func, false);
-  } else if (elem.attachEvent) {// OLD IE DOM 
+  } else if (elem.attachEvent) {// OLD IE DOM
     elem.attachEvent('on'+evnt, func);
   } else { // No much to do
     elem[evnt] = func;
   }
 };
 
+AdapterJS.renderNotificationBar = function (text, buttonText, buttonLink) {
```

Comment:
Any reason why this was moved here?
This function only makes sense when you're in a non-native-webrtc browser.